django/db/models/sql/query.py

```diff
@@ -1000,9 +1000,12 @@ def prepare_lookup_value(self, value, lookups, can_reuse, allow_joins=True):
         # Interpret '__exact=None' as the sql 'is NULL'; otherwise, reject all
         # uses of None as a query value.
         if value is None:
-            if lookups[-1] not in ('exact', 'iexact'):
-                raise ValueError("Cannot use None as a query value")
-            return True, ['isnull'], used_joins
+            if lookups[-1] not in self.query_terms:
```

Comment:
`self.query_terms` is removed as of af69f14e7ba06bb7dee257ec02b885f4828c77ff. It's probably not a good idea to use `QUERY_TERMS` as a replacement considering it's only used in one legacy spot in the admin. I'm not sure if this code is ideal -- at least a comment explaining it should be added.